content/kubermatic/master/advanced/oidc_auth/_index.en.md

```diff
@@ -33,92 +34,148 @@ Right after clicking on the button you will see a modal window where you can cop
+CAs certificates is included. Note that we expect that all certificates will be PEM encoded.
 
-```plaintext
---feature-gates={{ .Values.kubermatic.controller.featureGates }}  # must contain "OpenIDAuthPlugin=true"
---oidc-issuer-url={{ .Values.kubermatic.auth.tokenIssuer }}
---oidc-issuer-client-id={{ .Values.kubermatic.auth.issuerClientID }}
---oidc-ca-file={{ .Values.kubermatic.auth.caBundle }}
-```
+For example if the certificate used by your provider was issued by Let's Encrypt. You can visit
+[Let's Encrypt](https://letsencrypt.org/certificates) to download the necessary certificates and use the
+following command to prepare the bundle.
 
-{{% notice note %}}
-Note that `.Values.kubermatic.auth.caBundle` must contain OIDC provider's root CA certificates chain, see [Root CA certificates chain](#root-ca-certificates-chain) section that explains how to create the file.
-{{% /notice %}}
+```bash
+cat isrgrootx1.pem.txt lets-encrypt-x3-cross-signed.pem.txt > caBundle.pem
+```
 
-`conifg.json` file for `kubermatic-dashboard` must contain `"share_kubeconfig":true`.
-You can set it by changing the `kubermatic.ui.config` entry in the `values.yaml` file. Afterwards, [update KKP](#update-kubermatic).
+This bundle must then be copied verbatim into the `KubermaticConfiguration`.
 
-### Root CA Certificates Chain
+### Update KKP
 
-In order to verify OIDC provider's certificate in `kubermatic-controll-manager` when establishing TLS connection a public root CA certificate is required. Ideally the whole
-chain including all intermediary CAs certificates. Note that we expect that all certificates will be PEM encoded.
+After all values are set up, it's time to update the KKP master cluster. Update the `oauth` chart first:
 
-For example if the certificate used by your provider was issued by Let's Encrypt. You can visit [Let's Encrypt](https://letsencrypt.org/certificates) to download the necessary certificates
-and use the following command to prepare the bundle.
+**Helm 3**
 
 ```bash
-cat isrgrootx1.pem.txt lets-encrypt-x3-cross-signed.pem.txt > caBundle.pem
+helm --namespace oauth upgrade --install --wait --values values.yaml oauth charts/oauth/
 ```
 
-### Update KKP
+**Helm 2**
 
-After all values are set at the `values.yaml` the installed helm charts `kubermatic` and `oauth` need to get updated (at the master cluster):
+```bash
+helm upgrade --install --wait --timeout 300 --values values.yaml --namespace oauth oauth charts/oauth/
+```
+
+Now that the issuer is available, update the `KubermaticConfiguration`:
 
 ```bash
-helm upgrade --install --wait --timeout 300 --values values.yaml --namespace oauth oauth charts/kubermatic/oauth
-helm upgrade --install --wait --timeout 300 --values values.yaml --namespace kubermatic kubermatic charts/kubermatic/
+kubectl -n kubermatic apply -f kubermaticconfig.yaml
 ```
 
+After the operator has reconciled the KKP installation, OIDC auth should be available.
```

Comment:
I would replace `should` with `will become`.
Should is used for what is probable.